src/tests/compiler/statement/statementedNodeTests.ts

```diff
@@ -290,6 +297,13 @@ describe(nameof(StatementedNode), () => {
         it("should remove statements at the end default clause", () => {
             doFirstChildTest<DefaultClause>(defaultClause, [1, 2], "switch (x) {\n    default:\n        x = 0;\n}", SyntaxKind.DefaultClause);
         });
+
+        it("should remove statements in a Block", () => {
+            const {sourceFile, firstChild} = getInfoFromTextWithSyntax<Block>  ('function():number{const a = 1, b = true;}', SyntaxKind.Block);
+            expect(!!firstChild.getStatementByKind(SyntaxKind.VariableStatement)).equals(true);
```

Comment:
Could you change this to use syntax like `expect(something).to.not.be.undefined`?

Reply:
done